tests/integration/standard/test_connection.py

```diff
@@ -401,6 +402,28 @@ def setUp(self):
             raise unittest.SkipTest("Can't test asyncore with monkey patching")
         ConnectionTests.setUp(self)
 
+    def test_subclasses_share_loop(self):
+        class C1(AsyncoreConnection):
+            pass
+
+        class C2(AsyncoreConnection):
+            pass
+
+        cassandra.io.asyncorereactor._global_loop._cleanup()
+        cassandra.io.asyncorereactor._global_loop = None
+
+        clusterC1 = Cluster(connection_class=AsyncoreConnection)
```

Comment:
Should this be `connection_class=C1`, and the one below be `connection_class=C2`?

Reply:
Yeah, good catch :+1:

Reply:
Hmm, I just realized with test happens the same with the libev one, it doesn't fail on master. Still is a good test to have IMO, wdyt?

Reply:
Sorry :worried:  , I had forgot to push, this only makes sense if `connection_class=C1` of course. 
By the libev test I was referring to my last comment here: https://github.com/datastax/python-driver/pull/950 . This test is not failing in master because somewhere before this test is ran we call `cluster = Cluster(connection_class=AsyncoreConnection)` and `C1` and `C2` use `AsyncoreConnection`'s loop, instead of creating one each. (It's better exapined in the libev PR)